src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/string.txt

```diff
@@ -83,7 +83,7 @@ true
 >> Mid("foo", 0)
 #Error
 
-Mid(Text(1/0, "#.000"), 1)
+>> Mid(Text(1/0, "#.000"), 1)
```

Comment:
It turns out this test case actually hangs the interpreter!
Filed https://github.com/microsoft/Power-Fx/issues/195

Reply:
That probably also explains why this PR is failing the CI